frontend/catalyst/passes/builtin_passes.py

```diff
@@ -572,44 +577,43 @@ def to_ppr(qnode):
 
     **Example**
 
-    The ``to_ppr`` compilation pass can be applied as a dectorator on a QNode:
+    In this example the Clifford+T gates will be converted into PPRs.
 
     .. code-block:: python
 
         import pennylane as qml
+        from catalyst import qjit, measure
 
-        qml.capture.enable()
+        ppm_passes = [("PPM", ["to-ppr"])]
 
-        @qml.qjit(target="mlir")
-        @qml.transforms.to_ppr
+        @qjit(pipelines=ppm_passes, keep_intermediate=True, target="mlir")
         @qml.qnode(qml.device("null.qubit", wires=2))
         def circuit():
             qml.H(0)
             qml.CNOT([0, 1])
             qml.T(0)
-            return qml.expval(qml.Z(0))
-
-    For clear and inspectable results, use ``target="mlir"`` in the ``qjit`` decorator, ensure that
-    PennyLane's program capture is enabled, :func:`pennylane.capture.enable`, and call ``to_ppr``
-    from the PennyLane frontend (``qml.transforms.to_ppr``) instead of with
-    ``catalyst.passes.to_ppr``.
-
-    >>> print(qml.specs(circuit, level="all")()['resources'])
-    {
-        'No transforms': ...,
-        'Before MLIR Passes (MLIR-0)': ...,
-        'to-ppr (MLIR-1)': Resources(
-            num_wires=2,
-            num_gates=7,
-            gate_types=defaultdict(<class 'int'>, {'PPR-pi/4-w1': 5, 'PPR-pi/4-w2': 1, 'PPR-pi/8-w1': 1}),
-            gate_sizes=defaultdict(<class 'int'>, {1: 6, 2: 1}),
-            depth=None,
-            shots=Shots(total_shots=None, shot_vector=())
-        )
-    }
+            return measure(1)
```

Comment:
I am not sure how you want to handle the documentation here. This line will fail after turning capture on.

Reply:
yep that's fine :)